.gitignore

```diff
@@ -0,0 +1,24 @@
+# Compiled class file
```

Comment:
Do we really need everything in this file?
Please, optimize this gitignore file.